ui/desktop/app/controllers/scopes/scope/projects/targets/index.js

```diff
@@ -21,15 +21,36 @@ export default class ScopesScopeProjectsTargetsIndexController extends Controlle
 
   // =attributes
 
-  queryParams = ['search', 'page', 'pageSize'];
+  queryParams = ['search', { scopes: { type: 'array' } }, 'page', 'pageSize'];
 
+  allTargets;
   @tracked search;
+  @tracked scopeSearchTerm;
+  @tracked scopes = [];
+  @tracked selectedScopes = [];
   @tracked page = 1;
   @tracked pageSize = 10;
-  @tracked searchItems = ['Project 1', 'Project 2', 'Project 3'];
+
+  constructor() {
+    super(...arguments);
+    this.getAllTargets();
```

Comment:
I don't think this can be done here, aren't controllers singletons?

Reply:
Removed. Moved fetch back to the model hook.